packages/services/api/src/modules/alerts/providers/alerts-manager.ts

```diff
@@ -11,7 +12,6 @@ import { ProjectManager } from '../../project/providers/project-manager';
 import { Logger } from '../../shared/providers/logger';
 import type { ProjectSelector } from '../../shared/providers/storage';
 import { Storage } from '../../shared/providers/storage';
-import type { AlertsModule } from '../__generated__/types';
```

Comment:
now `import type` always on top? not sure why it was moved

Reply:
They're not always on top ([see `ProjectSelector` in same file](https://github.com/kamilkisiela/graphql-hive/pull/2173/files/acca56a7ecb3243291dba08b6624c8711c93a0c8#diff-230a1e435104ca30e66815972d61fb01469757b836f03d3cdb7495c09836c5d6L12))

I think the sort is just better now haha.